### PR TITLE
service called on wrong variable

### PR DESCRIPTION
**Pull request by sloba88**

### CODE REVIEW

File: cookbook/security/voters.rst [hunk 1]
```diff
@@ -226,7 +226,7 @@ from the authorization checker is called.
 
             $authChecker = $this->get('security.authorization_checker');
 
-            $this->denyAccessUnlessGranted('view', $post, 'Unauthorized access!');
+            $authChecker->denyAccessUnlessGranted('view', $post, 'Unauthorized access!');
```
Comment (stof) [inline cookbook/security/voters.rst]: this is wrong. the AuthorizationChecker does not have such method

Comment (sloba88) [inline cookbook/security/voters.rst]: My bad. But I still don't understand why is service instantiated in this example and then not used.

Comment (stof) [inline cookbook/security/voters.rst]: it is a left-over of the time when the controller shortcut was not used in the doc.

Comment (OskarStark) [inline cookbook/security/voters.rst]: so shall @sloba88 then remove the left-over @stof ?

**Comment by sloba88** [comment]
I'll just create another pull request for cleanup...

**Comment by OskarStark** [comment]
thank you!

please don't forget to reference this PR here

**Comment by xabbuh** [comment]
I don't see why we need this PR at all. The issue should be fixed at all when merging #5711, isn't it?

**Comment by xabbuh** [comment]
@sloba88 Closing here, feel free to reopen if you do not agree.

**Pull request closed without merging**